Example/Podfile

```diff
@@ -7,7 +7,7 @@ target 'Emission' do
                :subspecs => %w( Core RCTText RCTWebSocket RCTNetwork ),
                :inhibit_warnings => true
 
-  if ENV['ARTSY_STAFF_MEMBER'] != nil || ENV['CI'] != nil
+  if true #ENV['ARTSY_STAFF_MEMBER'] != nil || ENV['CI'] != nil
```

Comment:
it’s not that simple, though, env vars for one’s shell are usually defined in a shell specific file, and not all shells are compatible with each other. E.g. I use the fish shell and its files cannot be interpreted as (ba)sh.

iirc you have to use `~/.MacOSX/environment.plist` for GUI apps, not sure if that would be enough in our case. https://developer.apple.com/library/mac/documentation/MacOSX/Conceptual/BPRuntimeConfig/Articles/EnvironmentVars.html